<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="enabler">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="macros" stdset="0">
      <string>{&quot;DeviceName&quot;:&quot;BC1 I&quot;,&quot;write_pv&quot;:&quot;FBCK:FB04:LG01:S3DES&quot;,&quot;read_pv&quot;:&quot;BLEN:LI21:265:AIMAXCUS1H&quot;}</string>
     </property>
     <property name="filename" stdset="0">
      <string>read_write_tweak_device.py</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="macros" stdset="0">
      <string>{&quot;DeviceName&quot;:&quot;BC2 I&quot;,&quot;write_pv&quot;:&quot;FBCK:FB04:LG01:S5DES&quot;,&quot;read_pv&quot;:&quot;BLEN:LI24:886:BIMAXCUS1H&quot;}</string>
     </property>
     <property name="filename" stdset="0">
      <string>read_write_tweak_device.py</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="macros" stdset="0">
      <string>{&quot;DeviceName&quot;:&quot;Coll Left Jaw&quot;,&quot;write_pv&quot;:&quot;COLL:LI21:235:MOTR.VAL&quot;,&quot;read_pv&quot;:&quot;COLL:LI21:235:MOTR.RBV&quot;}</string>
     </property>
     <property name="filename" stdset="0">
      <string>read_write_tweak_device.py</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="macros" stdset="0">
      <string>{&quot;DeviceName&quot;:&quot;Coll Right Jaw&quot;,&quot;write_pv&quot;:&quot;COLL:LI21:236:MOTR.VAL&quot;,&quot;read_pv&quot;:&quot;COLL:LI21:236:MOTR.RBV&quot;}</string>
     </property>
     <property name="filename" stdset="0">
      <string>read_write_tweak_device.py</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMEmbeddedDisplay</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.embedded_display</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
